### PR TITLE
Broadpeaks

### PR DESCRIPTION
**Pull request by Maarten-vd-Sande**
**What problem is the PR solving / What's new?**
broad peaks can not make vis

**Checklist**
- [x] I made a PR to develop (not master)
- [ ] If applicable: I updated the docs
- [x] I updated the CHANGELOG
- [x] These changes are covered by the tests

### CODE REVIEW

File: seq2science/rules/qc.smk [hunk 1]
```diff
@@ -878,15 +878,16 @@ def get_peak_calling_qc(sample):
...
     if has_annotation(assembly):
         output.extend(expand("{genome_dir}/{{assembly}}/{{assembly}}.annotation.gtf", **config))  # added to be unzipped
         if config.get("deeptools_qc"):
             output.extend(expand("{qc_dir}/plotProfile_gene/{{assembly}}-{peak_caller}.tsv", **config))
-        if get_ftype(list(config["peak_caller"].keys())[0]) == "narrowPeak":
+        if narrowpeak_used:
             output.extend(expand("{qc_dir}/chipseeker/{{assembly}}-{peak_caller}_img1_mqc.png", **config))
             output.extend(expand("{qc_dir}/chipseeker/{{assembly}}-{peak_caller}_img2_mqc.png", **config))
-    if config.get("deeptools_qc"):
+    if config.get("deeptools_qc") and narrowpeak_used:
```
Comment (siebrenf) [inline seq2science/rules/qc.smk:890]: can these heatmaps be made if there is only 1 narrowpeak file in the assembly? 

If not, the DESeq2 plotcorrelations in qc.smk should have a similar checking system for the number of treps per assembly.

File: seq2science/rules/qc.smk [hunk 2]
```diff
@@ -878,15 +878,16 @@ def get_peak_calling_qc(sample):
 
     # deeptools profile
     assembly = treps.loc[sample, "assembly"]
+    narrowpeak_used = get_peak_ftype(list(config["peak_caller"].keys())[0]) == "narrowPeak"
```
Comment (siebrenf) [inline seq2science/rules/qc.smk:881]: `"narrowPeak" in [(get_peak_ftype(pc) for pc in list(config["peak_caller"].keys())]` maybe?

File: seq2science/rules/qc.smk (hunk 1 above)
Comment (Maarten-vd-Sande) [reply seq2science/rules/qc.smk:890]: I think so. It clusters on the peaks, not on the pattern. Not 100% sure though

File: seq2science/rules/qc.smk (hunk 2 above)
Comment (Maarten-vd-Sande) [reply seq2science/rules/qc.smk:881]: :+1:

**Review by siebrenf — APPROVED** [review]

**Pull request merged**